tests/Functional/Execution/ExecutionTest.php

```diff
@@ -84,17 +86,20 @@ public function testAcceptAnObjectWithNamedPropertiesAsArguments()
                 ])
         ]);
 
-        $rootValue    = 'rootValue';
-        $documentNode = parse('query Example { a }');
+        $rootValue = 'rootValue';
+        $document  = parse('query Example { a }');
 
         /** @var ExecutionResult $executionResult */
-        $result = Execution::execute($schema, $documentNode, $rootValue);
+        $result = executor()->execute($schema, $document, $rootValue);
```

Comment:
I would prefer `$this->execution = GraphQL::get(ExecutionInterface::class);` inside the `setUp` method.

Reply:
I agree, it's easier to change